src/main/java/subway/member/application/MemberService.java

```diff
@@ -28,6 +28,18 @@ public MemberResponse findMember(Long id) {
         return MemberResponse.of(member);
     }
 
+    public LoginMember findMemberByEmail(String email) {
+        Member member;
+
+        try {
+            member = memberDao.findByEmail(email);
+        } catch (DataAccessException ignored) {
+            return null;
+        }
```

Comment:
여기도 try ~ catch 문안에서 리턴해주는게 어떨까요?
```suggestion
       try {
           Member member = memberDao.findByEmail(email);  //FIXME  try~catch문 안에서 조회후 리턴가능하다.
            return new LoginMember(member.getId(), member.getEmail(), member.getAge());
        } catch (DataAccessException ignored) {
            return null;
        }